active/package-maintainers.md

```diff
@@ -0,0 +1,125 @@
+- No specific "Django Software Foundation" actions the group can take directly.
+
+Any request for funding or resources are made via existing processes outside of this group.
+This can be revised in the future as and when the group identifies specific recurring needs.
+
+## Membership
+
+- Chair: You?
+- Co-Chair: You?
+- Board Liaison (must be an active Board member; may be the same as Chair/Co-Chair): Thibaud Colas
+- Other members:
+  - You?
+
+## Eligibility
+
+Membership is open to all Individual Members of the Django Software Foundation.
+
+## How to join
+
+To join, members must express an interest in the #packages channel of the [Django Discord server](https://discord.gg/xcRH6mN4fa), or reach out to a current group member.
+
+Candidates will be invited to an upcoming Working Group meeting to discuss their interest and potential contributions, after which they will be voted in by current group members (50%+1).
+
+Members will be selected for the group based on their interest in the group’s goals and ability to contribute to them.
+
+### Membership terms
+
+Members join the group for a 6-month term. At the end of this term, they need to opt into staying involved to keep being a member of the group.
+
+If any member wishes to leave the group before the end of their term, they can do so without a vote.
+
+Members can propose a vote on removing a member from the working group. This needs 50%+1 agreement.
+
+## Budget
+
+No allocated budget, the group makes ad-hoc requests for funding or resources as needed.
+
+## Group communications
+
+The group communicates asynchronously via:
+
+- The public `#packages` channel on the Django Discord server.
+- Appropriate public sections of the [Django Forum](https://forum.djangoproject.com/).
+- GitHub issues and pull requests in a new `django/package-maintainers-wg` repository.
+- Collaborative documents set up in the Django Software Foundation’s Google Workspace.
+- A new private #package-maintainers-wg` channel on the Django Discord server for internal discussions.
+
+The group will also offer office hours for package maintainers every month via a video call.
+
+## Reporting
+
+The group will share highlights from current activities on a monthly basis via a forum post.
+
+## Appendix
+
+### Group activities
+
+As an illustration of the group’s remit, here are possible activities members could take part in.
+
+#### Support Django version compatibility efforts
+
+As Django regularly releases new versions, it’s an ongoing effort to ensure packages are compatible with new releases.
+
+Group members can support maintainers in this effort by:
+
+- Creating a shared calendar with Django release dates and deadlines.
+- Creating communication channels for package maintainers to coordinate compatibility changes for a specific release.
+- Recommending appropriate automation, such as [django-upgrade](https://github.com/adamchainz/django-upgrade) or how to set up Django pre-releases in Continuous Integration.
+
```

Comment:
```suggestion
- Proposing additions to the Django release notes with recommendations on how to handle deprecations.

```

I know that the `get_query_set` to `get_queryset` rename caused a lot of pain for some 3rd party package maintainers. Some guidance is better than none, and this working group could be a good place to discuss these concerns.